components/kflogin/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
I'm on the line about having this stock test, if we're never going to run it. I'm assuming any further developments here will make sure the app loads in the browser, which does more than what this file tests.